muxnect/server.py

```diff
@@ -98,7 +97,7 @@ def handle_request(window_name):
         except LibTmuxException:
             pass
 
-    return '200'
+    return '', 200
```

Comment:
Seems like you want to respond with a simple response body but a status code of 200. For sure you could technically send any string as the response and flask will make a 200 out of it, but IMHO (and let me know what you think) the most clean way, according to the Zen of Python, is explicitly returning a tuple containing the empty response and status code.